src/codegen/object.rs

```diff
@@ -47,6 +47,24 @@ pub fn generate(
             )?;
         }
 
+        if !analysis.builder_properties.is_empty() {
+            // generate builder method that returns the corresponding builder
+            let builder_name = format!("{}Builder", analysis.name);
+            writeln!(
+                w,
+                "
+            // rustdoc-stripper-ignore-next
+            /// Creates a new builder-style object to manufacture a `{name}`
+            /// This method returns an instance of `{builder_name}` which can be used to create a `{name}`.
```

Comment:
These need to be ``[`{foo}`]`` to get nice clickable links, or is that too verbose when the context/scope is already pretty clear?

Reply:
I would add a link here just so we are consistent with the rest of the builder docs I added before. It might be a bit verbose though I guess